jetbrains-toolbox.sh

```diff
@@ -32,8 +32,8 @@ fi
 
 chmod -R +rwx ${DIR}
 touch ${DIR}/jetbrains-toolbox.sh
-echo "#!/bin/bash" >> $DIR/jetbrains-toolbox.sh
-echo "$DIR/jetbrains-toolbox" >> $DIR/jetbrains-toolbox.sh
+echo '#!/bin/bash' >> $DIR/jetbrains-toolbox.sh
```

Comment:
I changed this because I noticed that my terminal was spewing out `bash: !/bin/bash: event not found`, I can revert this if it's an issue